spec.md

```diff
@@ -3237,14 +3237,14 @@ There are some important things for the SP and CO to consider here:
 
 - It is possible that the gRPC stream gets interrupted for arbitrary reasons beyond
   the control of either the SP or the CO.
-  The SP will get an error when writing to the stream and must abort its transmission.
+  The SP will get an error when writing to the stream and MUST abort its transmission.
   The CO will receive an error in its language specific idiom, which
   will be different from the language specific idiom for the end of a gRPC stream.
 
 - In all circumstances where the CO receives an error when reading from the gRPC stream
-  it may attempt to **continue** the operation by re-sending its request message but with a
-  starting_offset adjusted to a byte position beyond that of any metadata already received.
-  The SP must always examine the starting_offset and ensure that metadata sent in
+  it MAY attempt to **continue** the operation by re-sending its request message but with a
+  starting_offset adjusted to the NEXT byte position beyond that of any metadata already received.
+  The SP MUST always examine the starting_offset and ensure that metadata sent in
```

Comment:
nit: can we write `starting_offset` instead (wrapped in ticks) to clearly call out the field name? I realize that there are probably other places in the spec that don't do this right (prior to this PR) - and we should clean those up too, at some later point.